README.md

```diff
@@ -89,3 +89,11 @@ Go to `crystal/`
 ### .NET/TPL
 
 Build the solution with VS2015. Windows only :(
+
+### Rust (with [coroutine-rs](https://github.com/rustcc/coroutine-rs))
+
+```bash
+cd ./rust-coroutine
+cargo build --release
```

Comment:
Thanks!
I just wanted to make that explicit so noone would include compilation time in his benchmarks.